### PR TITLE
docs(gltf_extras): added small description of the newly added gltf_extras

### LINKED ISSUE #1329: Write release notes for PR #13453: add handling of all missing gltf extras: scene, mesh & materials
https://github.com/bevyengine/bevy/pull/13453 needs release notes for the upcoming Bevy release!

Please reply below if you'd like to write these notes. 
While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!

------

Release notes should:

1. Clearly motivate the change.
2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.

We can help you revise the release notes: a rough draft alone is incredibly useful :)
Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.

To submit your release notes, modify `..\release-content\0.14\release-notes\13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md` and submit a PR.
In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.

### PR DESCRIPTION
**Pull request by kaosat-dev**
closes #1329

### CODE REVIEW

File: release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md [hunk 1]
```diff
@@ -1,4 +1,32 @@
...
-<!-- https://github.com/bevyengine/bevy/pull/13453 -->
 
-<!-- TODO -->
+The gltf file format allows passing additional user defined metadata in the *extras* properties, and
+in addition to the gltf extras at the primitive/node level , Bevy now has specific GltfExtras for:
+- scenes: **SceneGltfExtras** injected at the scene level if any
+- meshes: **MeshGltfExtras**, injected at the mesh level if any
+- materials: **MaterialGltfExtras**, injected at the mesh level if any: ie if a mesh has a material that has gltf extras, the component will be injected there.
+
+You can now easilly query for these specific extras
```
Comment (alice-i-cecile) [inline release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md:8]:
```suggestion
You can now easily query for these specific extras
```

File: release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md [hunk 2]
```diff
@@ -1,4 +1,32 @@
-<!-- add handling of all missing gltf extras: scene, mesh & materials -->
-<!-- https://github.com/bevyengine/bevy/pull/13453 -->
 
-<!-- TODO -->
+The gltf file format allows passing additional user defined metadata in the *extras* properties, and
```
Comment (alice-i-cecile) [inline release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md:2]:
```suggestion
The glTF 3D model file format allows passing additional user defined metadata in the *extras* properties, and
```

File: release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md [hunk 3]
```diff
@@ -1,4 +1,32 @@
...
+in addition to the gltf extras at the primitive/node level , Bevy now has specific GltfExtras for:
```
Comment (alice-i-cecile) [inline release-content/0.14/release-notes/13453_add_handling_of_all_missing_gltf_extras_scene_mesh__materi.md:3]:
```suggestion
in addition to the glTF extras at the primitive/node level , Bevy now has specific GltfExtras for:
```

**Comment by kaosat-dev** [comment]
You are fast @alice-i-cecile  :D I was about to make the changes you mentioned, but you beat me to it :)

**Pull request merged**